src/preferences/dialog/dlgprefbroadcast.cpp

```diff
@@ -126,9 +126,9 @@ DlgPrefBroadcast::DlgPrefBroadcast(QWidget *parent, UserSettingsPointer _config)
     comboBoxEncodingFormat->setCurrentIndex(tmp_index);
 
     // Encoding channels combobox
-    comboBoxEncodingChannels->addItem(tr("Stereo"), BROADCAST_CHANNELS_STEREO);
+    comboBoxEncodingChannels->addItem(tr("Automatic"), BROADCAST_CHANNELS_AUTOMATIC);
```

Comment:
Isn't this box obsolete now? 
Maybe it could replaces with just a feedback field that indicates the resulting channel count.